pkg/daemon/daemon.go

```diff
@@ -261,7 +271,15 @@ func cmdRun(p *ptpProcess) {
 		for scanner.Scan() {
 			output := scanner.Text()
 			fmt.Printf("%s\n", output)
-			extractMetrics(p.name, p.iface, output)
+			out := fmt.Sprintf("%s\n", output)
+			if stdoutToSocket {
+				_, err := c.Write([]byte(out))
```

Comment:
if the sidecar pod crash/get restarted will the connection continue to work?

Reply:
you are right, it won't connect again. end up as broken pipe.
 I don't have a reconnect logic. I should implement reconnect here.

Reply:
done